cookbook/testing.md

```diff
@@ -127,31 +124,31 @@ const useCounterStore = defineStore('counter', {
 })
 ```
 
-Since the store is named _"counter"_, you need to add a matching object to `initialState`:
+Dado que el almacén se llama _"contador"_, es necesario añadir un objeto coincidente a `initialState`:
```

Comment:
I believe "counter" should not be translated here since is the name of the store. Do you agree?

Reply:
done